programmers/실패율.js

```diff
@@ -0,0 +1,22 @@
+function solution(N, stages) {
+   const answers = [];
+   
+   let totalNum = stages.length;
+   let count = 0;
+   let failRate = 0;
```

Comment:
컨벤션 신경써주시고 count와 failRate는 초기값을 선언하지 않아도 가능해요
```suggestion
   const answers = [];
   let totalNum = stages.length;
   let count;
   let failRate;
```

Reply:
let 같은 경우에는 초기값을 선언하지 않아도 되지만 되도록 선언해주는 것이 좋다고 하더라고요 :)